curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-markdown-previewer.md

```diff
@@ -8,7 +8,7 @@ dashedName: build-a-markdown-previewer
 
 # --description--
 
-**Objective:** Build an app that is functionally similar to this: <a href="https://codepen.io/freeCodeCamp/full/GrZVVO" target="_blank" rel="noopener noreferrer nofollow">https://codepen.io/freeCodeCamp/full/GrZVVO</a>.
+**Objective:** Build an app that is functionally similar to this: <a href="https://markdown-previewe.freecodecamp.rocks/" target="_blank" rel="noopener noreferrer nofollow">https://markdown-previewe.freecodecamp.rocks/</a>.
```

Comment:
this link doesn't work — I mistyped it, but please test what you are doing.

```suggestion
**Objective:** Build an app that is functionally similar to this: <a href="https://markdown-previewer.freecodecamp.rocks/" target="_blank" rel="noopener noreferrer nofollow">https://markdown-previewer.freecodecamp.rocks/</a>.
```

Reply:
> this link doesn't work — I mistyped it, but please test what you are doing.

Sorry... Actually all links were redirecting me then a discord maintainer told me that those links are not deploying yet. That's why... :⁠^⁠)

I changed the link now .. 

Please review it @ieahleen